### ISSUE TITLE
Add option to restore previous version of text file from history.

### ISSUE THREAD

**Issue by Alphalpha007** [mpcjanssen/simpletask-android#493]

**Comment by mpcjanssen** [comment]
You can already restore a file by first emptying the current list and then sharing the history item to Simpletask. It might be good to add a button for it though.

**Comment by Alphalpha007** [comment]
True it's possible. That's not obvious at all though.

**Comment by mpcjanssen** [comment]
Agreed,

**Comment by smichel17** [comment]
Agreed, particularly since the current workflow is actually to first delete everything from your list (otherwise you get duplicates. This is annoying, from experience), then share a version from the history to Simpletask.

**Comment by smichel17** [comment]
+1 for a button

>You can already restore a file **by first emptying the current list** and then sharing the history item to Simpletask

This is the part that is unintuitive, and if you don't do it you end up in versioning hell.
